src/core/server/app/middleware/passport/strategies/verifiers/sso.spec.ts

```diff
@@ -28,55 +28,6 @@ describe("SSOUserProfileSchema", () => {
       id: "id",
       email: "email",
       username: "username",
-      avatar: "avatar",
-    };
-
-    expect(validate(SSOUserProfileSchema, profile)).toEqual(profile);
-    expect(isSSOToken({ user: profile })).toEqual(true);
-  });
-
-  it("allows an empty avatar", () => {
```

Comment:
Why were these tests removed? Are there equivalent tests that could be written that match the new updates?

Reply:
These tests were removed as I removed the `displayName` and `avatar` fields for now.